$OpenBSD: patch-configure_ac,v 1.1 2013/09/25 07:56:57 sthen Exp $

quick hack for compat with giflib 5.0+; emacs is actually checking for "giflib
newer than 4.1.0b1" by using a check for a function which was added then (see
http://lists.gnu.org/archive/html/emacs-devel/2002-08/msg00417.html) but
that function has since been removed.

--- configure.ac.orig	Thu Sep 19 14:05:18 2013
+++ configure.ac	Thu Sep 19 14:05:37 2013
@@ -2661,7 +2661,7 @@ if test "${HAVE_X11}" = "yes" && test "${with_gif}" !=
   AC_CHECK_HEADER(gif_lib.h,
 # EGifPutExtensionLast only exists from version libungif-4.1.0b1.
 # Earlier versions can crash Emacs.
-    [AC_CHECK_LIB(gif, EGifPutExtensionLast, HAVE_GIF=yes, HAVE_GIF=maybe)])
+    [AC_CHECK_LIB(gif, EGifPutExtensionBlock, HAVE_GIF=yes, HAVE_GIF=maybe)])
 
   if test "$HAVE_GIF" = yes; then
     LIBGIF=-lgif
